src/app/public/modules/country-field/country-field.component.ts

```diff
@@ -185,17 +185,18 @@ export class SkyCountryFieldComponent implements ControlValueAccessor, OnDestroy
         this.ngControl.control.markAsPristine();
       }
 
+      if (this.isInitialChange) {
```

Comment:
You can always set this to `false`. There's no need for the `if` statement here.

Reply:
Good call. Removed